db/event_manager_test.go

```diff
@@ -2,6 +2,8 @@ package db
 
 import (
 	"fmt"
+	"github.com/couchbase/sync_gateway/base"
+	"github.com/stretchr/testify/assert"
```

Comment:
The standard practice with go is to separate the standard library imports from non-standard library imports, with an empty line between the two sets.

Reply:
Thank you Adam, I did a quick formatting on the entire file initially. It is fixed as per the standard.